components/clp-py-utils/clp_py_utils/sql_adapter.py

```diff
@@ -104,6 +108,16 @@ def create_connection(self, disable_localhost_socket_connection: bool = False):
         else:
             raise NotImplementedError
 
+    def create_root_mariadb_connection(self, disable_localhost_socket_connection: bool = False):
```

Comment:
if we have to add this method
```suggestion
    def create_root_mariadb_connection(self, disable_localhost_socket_connection: bool = False) -> mariadb.Connection:
```